spec/abilities/external_training_ability_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+require 'spec_helper'
+
+describe ExternalTrainingAbility do
+  let(:user) { role.person }
+  let(:user_group) { role.group }
+
+  subject { Ability.new(user.reload) }
+
+  let(:external_training) { Fabricate(:external_training, person: person) }
+
+  context 'with layer_and_below_full' do
+    let(:role) { Fabricate(Group::Geschaeftsstelle::Mitarbeiter.name.to_sym, group: groups(:geschaeftsstelle)) }
+
+    context 'in same layer' do
+      let(:person) { Fabricate(Group::Geschaeftsstelle::MitarbeiterLesend.name.to_sym, group: groups(:geschaeftsstelle)).person }
+
+      it 'can create and destroy' do
+        is_expected.to be_able_to(:create, external_training)
```

Comment:
die `subject is_expected.to` syntax ist imho alt und deprecated, neu wäre ` expect(ability).to`